.travis.yml

```diff
@@ -51,14 +51,15 @@ install:
           # want cmake newer than by default in travis osx
           brew upgrade cmake
           export CMAKE_FRAMEWORK_PATH="$CMAKE_FRAMEWORK_PATH:$(brew --prefix qt5)"
+          ADDITIONAL_LINK_ARGS="-stdlib=libc++ -mmacosx-version-min=10.7"
         fi
 script:
     - |
         if [ -z "$DEBUILD" ]
             then
             mkdir build
             cd build
-            cmake $CMAKE_ARGS .. || exit 1
+            cmake $CMAKE_ARGS -DADDITIONAL_LINK_FLAGS="$ADDITIONAL_LINK_ARGS" .. || exit 1
```

Comment:
you probably going to do it eventually but here is a reminder anywho. squash this with https://github.com/kvirc/KVIrc/pull/2079/commits/23bf81d1d26c3ac3370f1fdec7e0e5a71607b0bd